### PR TITLE
CAL-156 added CHANGES.md to track changes between versions

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds a change log to track differences between versions. 

#### Who is reviewing it?

@mcalcote 
@jrnorth 
@emanns95
@dcruver 

#### Choose 2 committers to review/merge the PR.

@jlcsmith
@pklinef

#### How should this be tested?
test links

#### What are the relevant tickets?

[CAL-156](https://codice.atlassian.net/browse/CAL-156)
[DDF-2510](https://codice.atlassian.net/browse/DDF-2510)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+### API CHANGES
+
+- None in this version. Initial release.
+
+### BUG FIXES
+
+- None in this version. Initial release.
+	- [Jira view of all issues resolved](https://codice.atlassian.net/issues/?jql=project%3D%22Codice%20Alliance%22%20AND%20type%20%3D%20Bug%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20%3D%20Alliance-0.2%20ORDER%20BY%20priority) in this version.
+### KNOWN ISSUES
```
Comment (emanns95) [inline CHANGES.md]: This appears on the same line as above

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- [CAL-32](https:/codice.atlassian.net/browse/CAL-32) Support to view NITF imagery as an image overview and have a preview of the full resolution image
+- [CAL-72](https:/codice.atlassian.net/browse/CAL-72) An original serialization of an NSILI response in the metacard metadata field so that I can ensure no loss of information
+- [CAL-7](https:/codice.atlassian.net/browse/CAL-7) A STANAG 4559 compliant input transformer to support NSIL type specific data models
+- [CAL-6](https:/codice.atlassian.net/browse/CAL-6) A STANAG 4559 input transformer so that I can ingest 4559 metadata into DDF
+- [CAL-8](https:/codice.atlassian.net/browse/CAL-8) Create a Codice DDF Alliance Distribution
+- [CAL-2](https:/codice.atlassian.net/browse/CAL-2) A STANAG 4559 simulator
+- [CAL-33](https:/codice.atlassian.net/browse/CAL-33) An image chipping service
+- [CAL-88](https:/codice.atlassian.net/browse/CAL-88) Updated ImagingChipActionProvider implements MultiActionProvider
+- [CAL-43](https:/codice.atlassian.net/browse/CAL-43) improve metacard retry logging in alliance-mpegts-stream
```
Comment (emanns95) [inline CHANGES.md]: None of these links are working. Possible due to https:/ missing a slash?

Comment (ricklarsen) [inline CHANGES.md]: yep. fixed.

**Comment by ricklarsen** [comment]
:shipit: ?

**Pull request merged**